src/openrct2/object/ObjectManager.cpp

```diff
@@ -147,7 +147,7 @@ class ObjectManager final : public IObjectManager
         log_verbose("%u / %u new objects loaded", numNewLoadedObjects, requiredObjects.size());
     }
 
-    void UnloadObjects(const rct_object_entry* entries, size_t count) override
+    void UnloadObjects(const std::vector<rct_object_entry> entries, size_t count) override
```

Comment:
This is doing a copy of the vector. Make it a `const std::vector<....> &` that way only the reference is taken and the expensive copy is not created.